### PR TITLE
DON'T MERGE: Part of [SAIC-241]: create temp dir for image processing step(unfinished, because need to setup ceph)

### PR DESCRIPTION
**Pull request by realsystem**

### CODE REVIEW

**Comment by Wasapl** [comment]
Build finished.
Unittest success

**Comment by Wasapl** [comment]
Unittest success

File: cloudferrylib/os/actions/load_compute_image_to_file.py [hunk 1]
```diff
@@ -17,10 +19,14 @@
 class LoadComputeImageToFile(action.Action):
     def run(self, info=None, **kwargs):
         cfg = self.cloud.cloud_config.cloud
+        temp_dir_name = tempfile.mkdtemp(prefix='image_dir_')
```
Comment (roman-verchikov) [inline cloudferrylib/os/actions/load_compute_image_to_file.py]: `temp_dir_name` dir must be deleted once not needed

**Comment by Wasapl** [comment]
Build finished.
Functional test success

**Comment by Wasapl** [comment]
Functional test success

**Comment by Wasapl** [comment]
Build finished.
Unittest success

**Comment by Wasapl** [comment]
Unittest success

**Comment by Wasapl** [comment]
Build finished.
Functional test failure

**Comment by Wasapl** [comment]
Functional test failure

**Comment by Wasapl** [comment]
Build finished.
Unittest success

**Comment by Wasapl** [comment]
Unittest success

**Comment by Wasapl** [comment]
Build finished.
Functional test failure

**Comment by Wasapl** [comment]
Functional test failure

**Pull request closed without merging**